sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -61,44 +62,70 @@ const deserializationContentTypes = {
   ]
 };
 
+/**
+ * Provides configuration options for AppConfigurationClient
```

Comment:
```suggestion
 * Provides configuration options for AppConfigurationClient.
```

Reply:
Fixed. Also, there was another one beneath it. We have become period-complete.